src/app/tx/[tx]/page.tsx

```diff
@@ -0,0 +1,176 @@
+'use client';
+import { L2ToL1MessageStatus } from '@arbitrum/sdk';
+import dynamic from 'next/dynamic';
+import { WagmiProvider } from '../../../components/WagmiProvider';
+import {
+  getL1ToL2MessagesAndDepositMessages,
+  getL1TxnReceipt,
+  getL2ToL1Messages,
+  receiptStateToDisplayableResult,
+} from '../../../lib';
+import {
+  L1ToL2MessagesAndDepositMessages,
+  L2ToL1MessageData,
+  L2TxnStatus,
+  ReceiptRes,
+  ReceiptState,
+} from '../../../types';
+
+const MessageDisplays = dynamic(
+  // @ts-expect-error Server Component
```

Comment:
Why is this ignore needed in case of `MessageDisplays` but not any other dynamic imports later?